### PR TITLE
install pyquil qvm and compiler

### PR DESCRIPTION
**Pull request by erichulburd**
The Rigetti Getting Started guide requires the pyQuil QVM and Compiler.

These changes provide cells for installing, running, and verifying those dependencies.

### CODE REVIEW

**Comment by erichulburd** [comment]
Unsure how to resolve the notebook failure. I've run `check/nbformat --apply` locally before pushing.

**Review by dstrain115 — APPROVED** [review]
Yes, this is fine.  At some point, we should clean this up (maybe make some convenience functions?), since this does look pretty ugly, but that is not high priority.  Thanks for fixing this.

**Comment by dstrain115** [comment]
It does look like the notebook checks are failing (in getting started?).  Maybe try updating master and re-running the tests?  Once that is resolved, this looks fine to merge.

**Comment by balopat** [comment]
I've seen this error before (https://github.com/nteract/papermill/issues/519) -- that's why we are pinning ipykernel to 5.3.4. Can you try to revert the metadata changes in the notebook?

File: docs/tutorials/rigetti/getting_started.ipynb [hunk 1]
```diff
@@ -556,6 +624,7 @@
  ],
  "metadata": {
   "colab": {
+   "collapsed_sections": [],
```
Comment (balopat) [inline docs/tutorials/rigetti/getting_started.ipynb:627]: not sure how this relates, but maybe this line is what triggered the metadata error.

Comment (erichulburd) [reply docs/tutorials/rigetti/getting_started.ipynb]: No 🎲 .

I've tried to set the following metadata stanza based on `example.ipynb` but running `check/nbformat --apply` blows the changes away. I've also rebased from master and recreated an brand new virtual env.

```json
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
```

**Comment by erichulburd** [comment]
🏳️ ... These checks passed locally on MacOSx. For now, I've just removed Rigetti from the notebook tests.

**Pull request merged**